src/plan_bot.py

```diff
@@ -139,8 +141,25 @@ def build_help_response_text():
 Have another question or run into any problems?  [Send a report](https://www.reddit.com/message/compose?to=WarrenPlanBotDev&subject=BotReport&message=Issue with help response).  
 """
 
+def build_advanced_response_text(plans, post):
+    return """I’m the WarrenPlanBot. If Elizabeth Warren has a plan, I can help you find it!
+
+If you start your message with `-p` or `--parent` like this:
```

Comment:
As noted below, I think we should skip the `-p` flag based on earlier conversation and we might want to change `--parent` to `--tell-parent`